tests/expectations/snapshots/snap_test_expectation_atomic_renderers.py

```diff
@@ -919,14 +899,14 @@
                 "schema": {"type": "array"},
                 "value": ["my_first_col", "my_second_col", "my_third_col"],
             },
-            "condition_parser": {"schema": {"type": "string"}, "value": None},
-            "ignore_row_if": {"schema": {"type": "string"}, "value": None},
+            "column_list_0": {"schema": {"type": "string"}, "value": "my_first_col"},
+            "column_list_1": {"schema": {"type": "string"}, "value": "my_second_col"},
+            "column_list_2": {"schema": {"type": "string"}, "value": "my_third_col"},
             "mostly": {"schema": {"type": "number"}, "value": 0.8},
             "mostly_pct": {"schema": {"type": "string"}, "value": "80"},
-            "row_condition": {"schema": {"type": "string"}, "value": None},
         },
         "schema": {"type": "com.superconductive.rendered.string"},
-        "template": "Values for given compound columns must be unique together, at least $mostly_pct % of the time: $column_list_0, $column_list_1, $column_list_2",
+        "template": "Values for given compound columns must be unique together, at least $mostly_pct % of the time: $column_list_0 $column_list_1 $column_list_2",
```

Comment:
The commas are removed here so that we can get consistent rendering behavior for arrays (other renderers don't add the commas). Styling is currently applied to the `$` params, so differentiating between params is easy.